### ISSUE TITLE
Path to reserved system folders work only once

### ISSUE THREAD

**Issue by francois333** [coreybutler/fenix#106]
If the path to the files to be served is a Windows reserved path (like 'Documents'), the server works the first time, but the next time it is booted, shows [object].

**Comment by coreybutler** [comment]
Probably won't fix since we're wrapping up the release candidate work for Fenix 3 (https://prewview.fenixwebserver.com).

**Comment by coreybutler** [comment]
Closing since Fenix 3 has launched. Please open a new issue if this occurs in the new version.
